### PR TITLE
fix: make it work on IE

### PR DESCRIPTION
**Pull request by flyingtof**
add build configuration with babel to transpile js to ES5 in order to
run on IE
fix extraction of text and css classes from html fragment

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -75,12 +75,16 @@ module.exports = {
...
+                }
             },
             {
                 test:/\.css$/,
                 loader: 'style-loader!css-loader'
             }
         ]
-    }
+    }/*,
+    devtool: "source-map"*/
```
Comment (jgiovaresco) [inline webpack.config.js]: This could be removed ?

File: src/Map.js [hunk 2]
```diff
@@ -2,11 +2,11 @@
 
 require('./style.css');
 const objectAssign = require('object-assign');
-const {isHTMLElement} = require('./utils/dom');
+const dom = require('./utils/dom');
```
Comment (jgiovaresco) [inline src/Map.js:5]: If we transpile this should not be necessary, no ?

**Review by jgiovaresco — COMMENTED** [review]
I'm surprise to see code modification when the commit message talks about the build

File: webpack.config.js [hunk 3]
```diff
@@ -75,12 +75,16 @@ module.exports = {
...
+                    presets: ['@babel/preset-env']
+                }
             },
             {
                 test:/\.css$/,
                 loader: 'style-loader!css-loader'
             }
         ]
-    }
+    }/*,
```
Comment (syl20b) [inline webpack.config.js]: I would remove the comment below

File: src/Map.js (hunk 2 above)
Comment (flyingtof) [reply src/Map.js:5]: it's just for code style consistency

File: webpack.config.js (hunk 1 above)
Comment (flyingtof) [reply webpack.config.js]: removed

File: webpack.config.js (hunk 3 above)
Comment (flyingtof) [reply webpack.config.js]: removed

**Review by jgiovaresco — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
-      "@babel/preset-env": "^7.1.0",
```
Comment (jgiovaresco) [inline package.json]: this change could be moved in the 1st commit ;)

Comment (flyingtof) [reply package.json]: change moved to first commit.

**Pull request merged**